### PR TITLE
added unit tests for validators

### PR DESCRIPTION
**Pull request by alexreed22496**

### CODE REVIEW

File: tests/Validators/SKUValidatorTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class skuValidatorTest extends TestCase
+{
+    public function testValidateSkuSuccess()
+    {
+        $sku = 'AAABBCCC11';
+        $result = skuValidator::validateSku($sku);
+        $this->assertEquals('AAABBCCC11', $result);
+    }
+
+    public function testValidateSKUFailure()
```
Comment (amandahrdmn) [inline tests/Validators/SKUValidatorTest.php:17]: consistent naming would be testValidateSkuFailure; same with below

File: tests/bootstrap.php [hunk 2]
```diff
@@ -1,3 +1,2 @@
 <?php
 require __DIR__ . '/../vendor/autoload.php';
-
```
Comment (turnerror) [inline tests/bootstrap.php:3]: lol

File: tests/Validators/StringValidatorTest.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+    public function testValidateExistsAndLengthSuccess()
+    {
+        $maxCharacterLength = 30;
+        $name = 'Loads of stocks';
+        $result = StringValidator::validateExistsAndLength($name, $maxCharacterLength);
+        $this->assertEquals('Loads of stocks', $result);
+    }
+
+    public function testValidateExistsAndLengthFailure()
```
Comment (turnerror) [inline tests/Validators/StringValidatorTest.php:18]: Isn't this a success check, this is how we want it to work.

File: tests/Validators/StringValidatorTest.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        $maxCharacterLength = 30;
+        $name = '';
+        $this->expectException(\Exception::class);
```
Comment (turnerror) [inline tests/Validators/StringValidatorTest.php:22]: This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.

File: tests/Validators/StringValidatorTest.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    public function testValidateExistsAndLengthFailure()
+    {
+        $maxCharacterLength = 30;
+        $name = '';
+        $this->expectException(\Exception::class);  
+        $result = StringValidator::validateExistsAndLength($name, $maxCharacterLength);    
+    }
+
```
Comment (turnerror) [inline tests/Validators/StringValidatorTest.php:25]: I think we are also missing a unit test to make sure that if our input is larger than the max characters we set, it also throws an exception.

File: tests/Validators/StockLevelValidatorTest.php [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        $stockLevel = '1234';
+        $result = StockLevelValidator::validateStockLevel($stockLevel);
+        $this->assertEquals('1234', $result);
+    }
+
+    public function testValidateStockLevelFailure()
+    {
+        $stockLevel = 'ABCD';
+        $this->expectException(\Exception::class);
```
Comment (turnerror) [inline tests/Validators/StockLevelValidatorTest.php:20]: This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.

File: tests/Validators/SKUValidatorTest.php [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        $sku = '@£$%';
+        $this->expectException(\Exception::class);
```
Comment (turnerror) [inline tests/Validators/SKUValidatorTest.php:20]: This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.

File: tests/Validators/PriceValidatorTest.php [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        $price = '12345';
+        $result = PriceValidator::validatePrice($price);
+        $this->assertEquals('12345', $result);
+    }
+
+    public function testValidatePriceFailure()
+    {
+        $price = 'ABCD';
+        $this->expectException(\Exception::class);
```
Comment (turnerror) [inline tests/Validators/PriceValidatorTest.php:20]: This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.

**Pull request merged**